packages/shared/src/components/ScrollFeedFiltersOnboarding.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement, useContext, useEffect } from 'react';
+import classNames from 'classnames';
+import { Button } from './buttons/Button';
+import { ScrollOnboardingVersion } from '../lib/featureValues';
+import { cloudinary } from '../lib/image';
+import AnalyticsContext from '../contexts/AnalyticsContext';
+import { AnalyticsEvent, TargetType } from '../lib/analytics';
+
+const GaussianBlur = (): ReactElement => {
+  return (
+    <div className="absolute w-52 h-52 rounded-full opacity-32 blur-2xl bg-theme-color-cabbage" />
+  );
+};
+
+const versionToContainerClassName: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'h-60',
+  [ScrollOnboardingVersion.V2]: 'h-[36rem]',
+};
+const versionToButtonClassName: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'w-[16.25rem]',
+  [ScrollOnboardingVersion.V2]: 'w-40',
+};
+const versionToButtonText: Record<ScrollOnboardingVersion, string> = {
+  [ScrollOnboardingVersion.V1]: 'Customize your feed',
+  [ScrollOnboardingVersion.V2]: 'Start',
+};
+
+interface ScrollFeedFiltersOnboardingProps {
+  version: ScrollOnboardingVersion;
+  onInitializeOnboarding: () => void;
+}
+export default function scrollFeedFiltersOnboarding({
+  version,
+  onInitializeOnboarding,
+}: ScrollFeedFiltersOnboardingProps): ReactElement {
+  const { trackEvent } = useContext(AnalyticsContext);
+  useEffect(() => {
+    trackEvent({
+      event_name: AnalyticsEvent.Impression,
+      target_type: TargetType.ScrollBlock,
+      target_id: version,
+    });
+  }, []);
+
+  return (
+    <div
+      className={classNames(
+        'flex fixed bottom-0 left-60 z-3 gap-16 justify-center items-center w-full pr-40',
+        versionToContainerClassName[version],
+      )}
+      style={{
+        background:
+          'linear-gradient(90deg, rgba(4, 19, 30, 1) 0%, rgba(4, 19, 30, 0.605797) 100%)',
+      }}
+    >
+      <div className="flex flex-col items-center">
+        {version === ScrollOnboardingVersion.V1 ? (
+          <GaussianBlur />
+        ) : (
+          <div className="flex relative justify-center items-center mb-10">
+            <GaussianBlur />
+            <img
+              className="w-[4.625rem] h-[4.625rem]"
+              src={cloudinary.feedFilters.supercharge}
+              alt="supercharge your feed"
+            />
+          </div>
+        )}
+        {version === ScrollOnboardingVersion.V2 && (
+          <h2 className="mb-6 font-bold typo-title2">
+            Supercharge your feed with
+            <br /> content you&apos;ll love reading!
+          </h2>
+        )}
+        <Button
```

Comment:
I think the user should be able to dismiss as well :/

Reply:
For now that's intended for the experiment.